src/main/java/com/petproj/moneychecker/mappers/ExpenseMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.petproj.moneychecker.mappers;
+
+import com.petproj.moneychecker.dto.ExpenseDTO;
+import com.petproj.moneychecker.entities.Expense;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface ExpenseMapper {
+    ExpenseMapper INSTANCE = Mappers.getMapper(ExpenseMapper.class);

```

Comment:
Are you sure you need this instance here? Will it work if we remove this INTANCE from here?